htdocs/projet/class/project.class.php

```diff
@@ -585,7 +585,11 @@ public function get_element_list($type, $tablename, $datefieldname = '', $dates
 		} elseif ($type == 'loan')
 		{
 			$sql = 'SELECT l.rowid, l.fk_user_author as fk_user FROM '.MAIN_DB_PREFIX."loan as l WHERE l.entity IN (".getEntity('loan').") AND l.fk_projet IN (".$ids.")";
-		} else {
+		} elseif ($type == 'shipping')
+		{
+            $sql = "SELECT e.rowid FROM ".MAIN_DB_PREFIX."expedition e JOIN ".MAIN_DB_PREFIX."element_element ee ON ee.fk_target=e.rowid AND ee.sourcetype='commande' AND ee.targettype='shipping' JOIN ".MAIN_DB_PREFIX."commande c on ee.fk_source=c.rowid WHERE c.".$projectkey." IN (".$ids.") AND e.entity IN (".getEntity($type).")";
```

Comment:
Even if a shipment is for a given order, each shipment has its own fk_projet field so we should use this one for the filter on project instead of using the one of order. Does the field fk_projet of your shipment is defined ?